driver/compenv.ml

```diff
@@ -106,14 +106,24 @@ type readenv_position =
 exception SyntaxError of string
 
 let parse_args s =
-  let args = String.split_on_char ',' s in
+  let args =
+    let len = String.length s in
+    if len = 0 then []
+    else
+      (* allow first char to specify an alternative separator in ":|; " *)
+      match s.[0] with
+      | ( ':' | '|' | ';' | ' ' ) as c ->
+         List.tl (String.split_on_char c s)
+      | _ -> String.split_on_char ',' s
+  in
   let rec iter is_after args before after =
     match args with
       [] ->
       if not is_after then
         raise (SyntaxError "no '_' separator found")
       else
-      (List.rev before, List.rev after)
+        (List.rev before, List.rev after)
+    | "" :: tail -> iter is_after tail before after
```

Comment:
Note: you are now allowing empty elements in the list.

Reply:
Yes, it was actually my intent.